docs/concepts/blocks.md

```diff
@@ -13,11 +13,113 @@ tags:
+
+Prefect Cloud and the Prefect server UI display a library of block types available for you to configure blocks that may be used by your flows.
+
+![Viewing the new block library in the Prefect UI](../img/ui/block-library.png)
+
+!!! tip "Blocks and parameters"
+    Blocks are useful for configuration that needs to be shared across flow runs and between flows.
+
+    For configuration that will change between flow runs, we recommend using [parameters](/concepts/flows/#parameters).
+
+## Prefect built-in blocks
+
+Prefect provides a broad range of commonly used, built-in block types. These block types are available in Prefect Cloud and the Prefect server UI and local installations of Prefect.
+
+| Block | Slug | Description |
+| --- | --- | --- |
+| [Azure](/concepts/filesystems/#azure) | `azure` | Store data as a file on Azure Datalake and Azure Blob Storage. |
+| [Date Time](/api-ref/prefect/blocks/system/#prefect.blocks.system.DateTime) | `date-time` | A block that represents a datetime |
+| [Docker Container](/api-ref/prefect/infrastructure/#prefect.infrastructure.DockerContainer) | `docker-container` | Runs a command in a container. |
+| [Docker Registry](/api-ref/prefect/infrastructure/#prefect.infrastructure.docker.DockerRegistry) | `docker-registry` | Connects to a Docker registry.  Requires a Docker Engine to be connectable. |
+| [GCS](/concepts/filesystems/#gcs) | `gcs` | Store data as a file on Google Cloud Storage. |
+| [GitHub](/concepts/filesystems/#github) | `github` | Interact with files stored on public GitHub repositories. |
+| [JSON](/api-ref/prefect/blocks/system/#prefect.blocks.system.JSON) | `json` | A block that represents JSON |
+| [Kubernetes Cluster Config](/api-ref/prefect/blocks/kubernetes/#prefect.blocks.kubernetes.KubernetesClusterConfig) | <span class="no-wrap">`kubernetes-cluster-config`</span> | Stores configuration for interaction with Kubernetes clusters. |
+| [Kubernetes Job](/api-ref/prefect/infrastructure/#prefect.infrastructure.KubernetesJob) | `kubernetes-job` | Runs a command as a Kubernetes Job. |
+| [Local File System](/concepts/filesystems/#local-filesystem) | `local-file-system` | Store data as a file on a local file system. |
+| [Microsoft Teams Webhook](/api-ref/prefect/blocks/notifications/#prefect.blocks.notifications.MicrosoftTeamsWebhook) | `ms-teams-webhook` | Enables sending notifications via a provided Microsoft Teams webhook. |
+| [Opsgenie Webhook](/api-ref/prefect/blocks/notifications/#prefect.blocks.notifications.OpsgenieWebhook) | `opsgenie-webhook` | Enables sending notifications via a provided Opsgenie webhook. |
+| [Pager Duty Webhook](/api-ref/prefect/blocks/notifications/#prefect.blocks.notifications.PagerDutyWebHook) | `pager-duty-webhook` | Enables sending notifications via a provided PagerDuty webhook. |
+| [Process](/concepts/infrastructure/#process) | `process` | Run a command in a new process. |
```

Comment:
This matches elsewhere in the docs - but I feel like the key thing with Process is the customizability.

Reply:
Good callout. I'll address this in a future PR.